src/app/domain/osReceive/models/OsReceiveCozyApp.ts

```diff
@@ -55,9 +56,16 @@ export interface AppForUpload {
   slug: string
 }
 
-export const fetchOsReceiveCozyApps = {
-  definition: Q('io.cozy.apps'),
-  options: {
-    as: 'io.cozy.apps/accept_from_flagship'
+export const fetchStackApp = (
+  slug: string
+): {
+  definition: QueryDefinition
+  options: QueryOptions
+} => {
+  return {
+    definition: Q('io.cozy.apps').getById(`io.cozy.apps/${slug}`),
```

Comment:
You should add a `select` to only get the `_id` for instance since you don't care about the object, you just care about the throw or not. 

(in that case, you should rename the `as` to reflect that you only get the `_id` field)